components/public-api-server/pkg/oidcservice/stub_client.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package oidcservice
```

Comment:
Currently, this doesn't seem to be generally usable only by the test, so I'd inline it in the test. It ensures that it can't be imported into production code, and helps discoverability.

Reply:
done